client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -84,22 +100,43 @@ export const InboundReturnEditModal = ({
     alertRef?.current?.scrollIntoView({ behavior: 'smooth' });
   };
 
+  const CancelButton = <DialogButton onClick={onClose} variant="cancel" />;
+  const BackButton = (
+    <DialogButton onClick={() => onChangeTab(Tabs.Quantity)} variant="back" />
+  );
+  const NextStepButton = (
+    <DialogButton
+      onClick={handleNextStep}
+      variant="next"
+      disabled={!lines.length}
+      customLabel={t('button.next-step')}
+    />
+  );
+  const OkButton = <DialogButton onClick={onOk} variant="ok" />;
+  const OkAndNextButton = (
+    <DialogButton
+      onClick={handleNextItem}
+      variant="next"
+      disabled={currentTab !== Tabs.Reason}
+    />
+  );
```

Comment:
Thought it was a bit easier to read to define all the possible buttons here, and render them conditionally below, rather than mixing the component definitions in with the conditional logic.